tests/resources/Github-Util.robot

```diff
@@ -34,4 +34,14 @@ Post Comment To Github Issue
     \   ${status}  ${result}=  Run Keyword And Ignore Error  Post  https://api.github.com/repos/vmware/vic/issues/${num}/comments?access_token\=%{GITHUB_AUTOMATION_API_KEY}  data={"body": "${comment}"}
     \   Exit For Loop If  '${status}'
     \   Sleep  1
-    Should Be Equal  ${result.status_code}  ${201}
\ No newline at end of file
+    Should Be Equal  ${result.status_code}  ${201}
+
+Check VMware Organization Membership
```

Comment:
Does this need to be used anywhere right now?

Reply:
nope, just a new potentially useful function